src/pages/jihye/Main/Main.scss

```
@@ -0,0 +1,320 @@
+  height: 100vh;
+  margin: 0 auto;
+  min-width: 780px;
+  max-width: 1100px;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+/*----------- nav ------------*/
+  nav {
+    width: 84%;
+    height: 70px;
+    min-width: 660px;
+    line-height: 30px;
+    padding: 10px 10px;
+    padding-bottom: 20px;
+    position: fixed;
+    z-index: 100;
+    top:0px;
+    left: 6%;
+    margin: 0 auto;
+    background-color: #fff;
+  /* #navLeft */
+    #navLeft {
+      display: inline-block;
+
+      li {
+        font-weight: normal;
+        font-family: 'Lobster', cursive;
+        font-size: 30px;
+        display: inline-block;
+        margin: 0 5px;
+      }
+      img {
+        width: 35px;
+      }
+    }
+  /* #navSearch */
+    #navSearch {
+      display: inline-block;
+      position: absolute;
+      top:15px;
+      left: 35%;
+
+      input {
+        width: 250px;
+        padding: 5px;
+        background-color: #f7f7f7;
+        border: 1px solid #dbdbdb;
+        border-radius: 5px;
+        text-align: center;
+      }
+      img {
+        position: absolute;
+        width: 15px;
+        top:8px;
+        left:50%;
+        margin-left: -30px;
+      }
+    }
+  /* #navRight */
+    #navRight {
+      display: inline-block;
+      position: absolute;
+      top: 15px;
+      right: 20px;
+
+      li{
+        display: inline-block;
+        padding: 0 5px;
+        margin: 0 5px;
+      }
+      
+      .naviRightImage {
+        img {
+          width: 30px;
+        }
+      }
+    } 
+  }
+
+/*------------ div #main ------------*/
+  #main {
+    width: 90%;
+    height: 100%;
+    margin: 0 auto;
+    padding: 8rem 1rem 0 3rem;
+    display: flex;
+    flex-direction: row;
+
+    .feeds {
+      width: 100%;
+      height: 100%;
+    }
+      article {
+        width: 100%;
+        border: 1px solid lightgray;
+      }
+    /*--- userSection ---*/
+        #userSection {
+          padding: 10px;
+          position: relative;
+          line-height: 25px;
+          font-weight: bold;
+          img {
+            width: 30px;
+            border-radius: 50%;
+            margin-right: 10px;
+          }
+          .more {
+            width: 20px;
+            position: absolute;
+            top: 15px;
+            right: 5px;
+          }
+        }
+    /*--- #userImage ---*/
+        #userImage{
+          text-align: center;
+          border-top: 1px solid lightgray;
+          border-bottom: 1px solid lightgray;
+          img {
+          width: 350px;
+          }
+        }
+    /*--- #guestSection ---*/
+        #guestSection {
+      /* .clickLike */
+          .clickLike {
+            padding: 5px;
+            position: relative;
+            img {
+              width: 30px;
+              padding: 0 3px;
+            }
+            img:nth-child(4) {
+              position: absolute;
+              top:5px;
+              right: 5px;
+            }
+          }
```

```suggestion
        #guestSection {
      /* .clickLike */
      
          .clickLike {
            padding: 5px;
            position: relative;
            
            img {
              width: 30px;
              padding: 0 3px;
            }
            
            img:nth-child(4) {
              position: absolute;
              top:5px;
              right: 5px;
            }
          }
```
선택자 바로위 한 줄은 개행처리 해줘서 코드의 가독성을 좀 더 챙겨주시면 좋겠습니다!